security/pkg/server/ca/monitoring.go

```diff
@@ -67,8 +66,7 @@ var (
 	)
 	certChainExpiryTimestamp = monitoring.NewGauge(
 		"citadel_server_cert_chain_expiry_timestamp",
-		"The unix timestamp, in seconds, when Citadel cert chain will expire. "+
-			"A negative time indicates the cert is expired.",
```

Comment:
Although this was the documentation, it was incorrect according to my testing.
So we can update the docs (as I have done here) and always record the expiration timestamp regardless of whether its expired, or we can keep the docs and update the behavior to record a negative if it is expired.

I think the recorded value should be the same regardless of whether of whether the cert has expired because the user can always just compare it to the current time.
Further, if you can just check the sign of `citadel_server_cert_chain_expiry_seconds` for a quick check of whether the cert chain has expired. 
I do wonder if this counts as a breaking change